case-server/src/main/java/com/powsybl/caseserver/dao/elasticsearch/CaseInfosDAOImpl.java

```diff
@@ -0,0 +1,84 @@
+/**
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.caseserver.dao.elasticsearch;
+
+import com.powsybl.caseserver.dao.CaseInfosDAO;
+import com.powsybl.caseserver.dto.CaseInfos;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import org.elasticsearch.index.query.QueryBuilders;
+import org.joda.time.DateTime;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.lang.NonNull;
+
+/**
+ * A class to implement metadatas transfer in the DB elasticsearch
+ *
+ * @author Slimane Amar <slimane.amar at rte-france.com>
+ */
+@ComponentScan(basePackageClasses = {CaseInfosRepository.class})
```

Comment:
A spring @service refers only to the business logic ?
It is a bean (explicity creation) which allows access to the database repository.
I don't know if it can be considered as a service.
I can rename the class to CaseInfosService